examples/policy/all-ethernet-up/current.yml

```diff
@@ -0,0 +1,26 @@
+---
+interfaces:
```

Comment:
Does it make sense to get the yamls from "github.com/nmstate/nmpolicy" ? at least shile nmpolicy is not from nmstatectl everywhere, this way if we add something at nmpolicy we know is suported at the nmstatectl version.

Reply:
Once this PR is merged. The `github.com/nmstate/nmpolicy` should be in archive state.